apiv2/go.mod

```diff
@@ -5,6 +5,8 @@ module github.com/open-edge-platform/infra-core/apiv2/v2
 
 go 1.24.6
 
+toolchain go1.24.9
```

Comment:
Is this toolchain version needed? Has it been included since the inventory version is being updated?